neuralpredictors/layers/readouts.py

```diff
@@ -956,6 +958,13 @@ def forward(self, x, sample=None, shift=None, out_idx=None):
                            if sample is True/False, overrides the model_state (i.e training or eval) and does as instructed
             shift (bool): shifts the location of the grid (from eye-tracking data)
             out_idx (bool): index of neurons to be predicted
+            multiplex (bool): if True, the neurons do not readout from their grid position,
+                            but from every position, i.e. every pixel, instead. Thus, each neuron is effectively copied
+                            to all positions in the image. The neuronal activity output matrix is then no longer
+                            (Batch, Neurons), but (Batch, Neurons x locations)
+            crop_edge_px (int): When multiplex is True, the response to all pixels that are cropped. Cropping
```

Comment:
could you include description about `collapse` and also update `multiplex` argument accordingly (about the shape of the output)

Reply:
Thanks, I've addressed it!